### PR TITLE
ZDC north/south energy

### PR DESCRIPTION
**Pull request by eumaka**
create module to extract zdc calibrated energy sums in north and south

### CODE REVIEW

**Comment by eumaka** [comment]
Need to add forgotten sqrt for r calculation, but will wait for this to go through first so as not to re-trigger Jenkins

**Comment by eumaka** [comment]
ah nvm, it turns out I already did :)

**Comment by pinkenburg** [comment]
You committed a lot of binary files which don't belong here (I assume your editor generated them?). Please remove them from this PR

**Comment by eumaka** [comment]
yep, thanks to vscode

**Comment by eumaka** [comment]
thanks Jin for killing the extra checks and sorry Chris for the expensive checks :)

File: offline/packages/zdcinfo/Zdcinfov1.h
Comment (pinkenburg) [inline offline/packages/zdcinfo/Zdcinfov1.h:1]: We should initialize the energy to NaN. {} initializes to zero which is a possible value

**Pull request merged**